src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -186,6 +192,8 @@ const std::shared_ptr<const Table>& SQLPipelineStatement::get_result_table() {
     return _result_table;
   }
 
+  _verify_lqp();
```

Comment:
why do this here? why not do this exactly where the LQP was created, i.e., `get_unoptimized_logical_plan()`. If there is a good reason for this being called here, please doc it.

Reply:
If this is verified in `get_unoptimized_logical_plan()` (like it is now) the verification doesn't take place when sending `CREATE TABLE t (...)` via the console twice. Then there is no new unoptimized LQP generated, because the PQP is already cached from the first execution.

Reply:
Yep, that sounds like a valid reason to put "_verify_lqp()" here. Please add a clarifying comment.